tests/test_cnn/test_build_layers.py

```diff
@@ -59,6 +59,9 @@ def test_build_conv_layer():
     assert layer.groups == kwargs['groups']
     assert layer.dilation == (kwargs['dilation'], kwargs['dilation'])
 
+    # some of sparse conv do not have arguments 'dilation' and 'group'
+    kwargs = dict(in_channels=4, out_channels=8, kernel_size=3)
```

Comment:
Other conv layers should test this. Therefore, we cannot simply change the kwargs but only pop out the group number for spconv layers

Reply:
The `SparseInverseConv2d` and `SparseInverseConv3d` [here](https://github.com/wHao-Wu/mmcv/blob/add-mmdet3d-ops-spconv/mmcv/ops/sparse_conv.py#L340) cannot take the `dilation` as input (although these classes inherit from the base class `SparseConvolution`, which can take the `dilation` as input). So there are some choices:

1. Pop out the `dilation` and `group` in kwargs;
2. Revise the interfaces of `SparseInverseConv2d` and `SparseInverseConv3d.